EIC-EA/EIC-community-D8
# FIX: layout for group members overview

### How to test

go to group member overview and check you have a correct layout like 
http://localhost:9000/?path=/story/bundles-group--group-member-private

- [ ] Groups members overview is ok on Storybook
- [ ] Groups members overview is ok on Localhost
